### PR TITLE
[docs]: add prev next links.

### LINKED ISSUE #507: [documentation]: add a way to navigate to next/prev page like we had on the old website.
I found it a bit annoying having to scroll all the way up to go to the next/prev page on docs I think we should have these arrows as we had on the old docs at the bottom of each page.  See the bottom of https://deploy-preview-1019--gitpod-website.netlify.app/docs/command-line-interface/

image

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #507 

This is how they look: 

image

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/609"></a>

### CODE REVIEW

File: src/routes/docs/menu.ts [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+  M("References", "references", [
+    // M(".gitpod.yml", "references/gitpod-yml"),
+    // M("Custom Docker image", "references/gitpod-dockerfile"),
+    // M("Architecture", "references/architecture"),
+    // M("Troubleshooting", "references/troubleshooting"),
+    M("Changelog", "references/changelog"),
+    M("Roadmap", "references/roadmap"),
+  ]),
+];
+const MENU_v1 = [
```
Comment (mikenikles) [inline src/routes/docs/menu.ts:83]: I don't think we need this anymore. Could you double check and remove it if it's unused?

File: src/components/docs/docs-content-layout.svelte [hunk 2]
```diff
@@ -4,7 +4,11 @@
   import FeedbackWidget from "./feedback-widget.svelte";
   import EditInGitpod from "../../components/docs/edit-in-gitpod.svelte";
   import docsCurrentSectionStore from "../../stores/docs-current-section";
+  import { page } from "$app/stores";
+  import { getMenuContext } from "../../routes/docs/menu";
+  import PrevNext from "./prev-next.svelte";
 
+  const menuCtx = getMenuContext($page.path);
```
Comment (mikenikles) [inline src/components/docs/docs-content-layout.svelte:11]: Let's move this to `src/components/docs/prev-next.svelte`. I don't think the `docs-content-layout.svelte` component needs to be aware of this code.

File: src/components/docs/prev-next.svelte [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+  import type { MenuContext } from "../../routes/docs/menu";
+  export let menuCtx: MenuContext;
+</script>
+
+<div class="flex justify-between">
+  {#if menuCtx.prev}
+    <a href={menuCtx.prev.path} title={`Prev: ${menuCtx.prev.title}`}> Prev </a>
```
Comment (mikenikles) [inline src/components/docs/prev-next.svelte:8]: 💡 How about we use the page title and an arrow?

Comment (nisarhassan12) [reply src/components/docs/prev-next.svelte:8]: Thanks, Mike for the suggestions indeed using an arrow indeed looks a lot better. 
image

Using the Page title does not work well on mobile when the titles are long.

image

Comment (nisarhassan12) [reply src/components/docs/prev-next.svelte]: Sorry I didn't look up the MAD specs earlier the following is how it looks according to them: 

image

**Review by mikenikles — APPROVED** [review]
Once the menu items are aligned with the latest in `main`, let's get this merged.

File: src/routes/docs/menu.ts [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+function M(title: string, path: string, subMenu?: MenuEntry[]): MenuEntry {
+  return {
+    title,
+    path: "/docs" + (path ? "/" + path : ""),
+    subMenu,
+  };
+}
+
+export const MENU: MenuEntry[] = [
```
[the remark this answers is not in the document]
Comment (nisarhassan12) [reply src/routes/docs/menu.ts:16]: Thanks. @mikenikles for catching this I just took the latest changes from main and added them manually to the file.

**Pull request merged**